src/components/BaseLayerSwitcher/BaseLayerSwitcher.js

```diff
@@ -152,8 +160,8 @@ function BaseLayerSwitcher({
               key={layer.key}
               className={`rs-base-layer-switcher-button ${openClass}${activeClass}`}
               role="button"
-              title={layerName}
-              aria-label={layerName}
+              title={t(layerName)}
```

Comment:
I was debating on whether to apply the t-function when declaring the layerName variable at the beginning of the loop and thereby calling it fewer times, but since I'm using that variable to create the activeClass variable I thought it was a bit dodgy (I would have to translate currentLayer.getName() as well), so I apply it every time I call layerName. Let me know if you agree.

Reply:
no do this here is easier